lib/errors.js

```diff
@@ -40,12 +40,21 @@ module.exports = {
     },
     MandatoryParamsNotFound: function(paramList) {
         this.name = 'MANDATORY_PARAMS_NOT_FOUND';
-        this.message = 'Some of the mandatory params weren\'t found in the request: ' + JSON.stringify(paramList);
+        this.message =
+            /*jshint quotmark: double */
+            "Some of the mandatory params weren't found in the request: " +
+            /*jshint quotmark: single */
+            JSON.stringify(paramList);
         this.code = 400;
     },
     UnsupportedType: function(expectedType) {
         this.name = 'UNSUPPORTED_TYPE';
-        this.message = 'The request content didn\'t have the expected type [' + expectedType + ' ]';
```

Comment:
This should be morer compact. Maybe using [%s] it could be done.

Reply:
Fixed 7657825